k-distribution/tests/regression-new/map-symbolic-tests/Makefile

```diff
@@ -1,7 +1,7 @@
 DEF=map-tests
 EXT=mtest
 TESTDIR=.
-KOMPILE_BACKEND=java
+KOMPILE_BACKEND=haskell
 KPROVE_FLAGS=--restore-original-names --no-alpha-renaming --warnings none
 
 include ../../../include/kframework/ktest.mak
```

Comment:
We should probably run these tests with both backends for now.

Reply:
Yep, I know. I just wanted to test quickly, but doesn't seem to be a good idea, making Jenkins too loaded. Will not do that later.

Reply:
Well you can also test locally instead. Just run `make` in this directory after building.

Reply:
Yeah, I thought that K Jenkins build is quicker than compiling the haskell backend in my machine, which was wrong.